open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/test/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/vrp/retrieval/flow/VRPConsentRetrievalUtilTest.java

```diff
@@ -0,0 +1,389 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
```

Comment:
Check here as well

ref: https://github.com/wso2/financial-open-banking/blob/main/open-banking-accelerator/components/com.wso2.openbanking.accelerator.gateway/src/main/java/com/wso2/openbanking/accelerator/gateway/cache/CertificateRevocationCache.java#L1

Reply:
THis seems not yet updated, can you check?

Reply:
Fixed in [02c291d]